jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java

```diff
@@ -53,4 +54,30 @@ public void positiveTest(String value) {
         properties.setProperty(SCREENS_FOLDER_PROPERTY.getName(), value);
         validateProperties(properties);
     }
+
+    @Test
+    public void crossPlatformPathTests() {
+        String chromePath = "";
+
+        String os = System.getProperty("os.name");
+        switch (os) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286)